app/src/utils/apiHelpers.test.tsx

```diff
@@ -0,0 +1,212 @@
+
+jest.mock("./apiResponse");
+
+beforeEach(() => {
+  jest.clearAllMocks();
+});
+
+describe("getHomePageData", () => {
+  it("creates response containing random number and all 7 lanes", async () => {
+    (apiResponse as jest.Mock).mockResolvedValueOnce(
+      Promise.resolve({
+        nyplAPI: {
+          response: {
+            counts: {
+              count: [
+                {
+                  uuid: { $: "test" },
+                  count_value: { $: "10" },
+                },
+              ],
+            },
+          },
+        },
+      })
+    );
+    const result = await getHomePageData();
+    expect(apiResponse as jest.Mock).toHaveBeenCalled();
+    expect([0, 1]).toContain(result.randomNumber);
+    expect(result.lanesWithNumItems.length).toEqual(7);
+    // Test uuid response doesn't match the collection uuid, so all of these should be 0.
+    expect(
+      result.lanesWithNumItems[0].collections[3].numberOfDigitizedItems
+    ).toEqual("0");
+  });
+});
+
+describe("getFeaturedItemData", () => {
+  it("creates response containing featuredItem and numDigitizedItems", async () => {
+    const result = await getFeaturedItemData();
+
+    // Fallback values:
+    expect(result.numberOfDigitizedItems).toEqual("875,861");
+    expect(result.featuredItem.imageID).toEqual(
+      defaultFeaturedItem.production.featuredItem.imageID
+    );
+  });
+});
+
+describe("getDivisionData", () => {
+  it("forms the correct request from params", async () => {
+    const result = await getDivisionData({
+      slug: "testSlug",
+      pageNum: 1,
+      perPage: 3,
+    });
+
+    expect(apiResponse as jest.Mock).toHaveBeenCalledWith(
+      `${process.env.API_URL}/api/v2/divisions/testSlug?page=1&per_page=3`
+    );
+  });
+  it("returns successful response", async () => {
+    (apiResponse as jest.Mock).mockResolvedValueOnce(
+      Promise.resolve({
+        headers: { status: "success", code: "200", message: "ok" },
+        summary: "divisions test",
+        divisions: [
+          {
+            name: "Billy Rose Theatre Division",
+            slug: "billy-rose-theatre-division",
+            collections: [],
+          },
+          {
+            name: "Carl H. Pforzheimer Collection of Shelley and His Circle",
+            slug: "carl-h-pforzheimer-collection-of-shelley-and-his-circle",
+            collections: [],
+          },
+        ],
+      })
+    );
+    const result = await getDivisionData({
+      pageNum: 1,
+      perPage: 2,
+    });
+    expect(result.divisions.length).toEqual(2);
+    expect(result).toHaveProperty("summary");
+  });
+});
+
+describe("getNumDigitizedItems", () => {
+  it("returns the correct numDigitizedItems", async () => {
+    (apiResponse as jest.Mock).mockResolvedValueOnce(
+      Promise.resolve({
+        count: {
+          $: 78,
+        },
+      })
+    );
+    const result = await getNumDigitizedItems();
+
+    expect(result).toEqual("78");
```

Comment:
I like the expectation you have on line 72. Can you add a similar one here to test/verify the URL that is called?